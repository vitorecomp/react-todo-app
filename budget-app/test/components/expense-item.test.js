import React from 'react'
import { shallow } from 'enzyme'

import ExpenseItem from '../../src/components/expense-item'
import expenses from '../fixtures/expense.fixtures'


test('should render Expense item correctly, with expense item', () => {
	const wrapper = shallow(<ExpenseItem {...expenses[0]} />)
	expect(wrapper).toMatchSnapshot();
})

