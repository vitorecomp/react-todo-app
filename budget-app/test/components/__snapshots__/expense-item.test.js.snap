// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense item correctly, with expense item 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Gun
    </h3>
  </Link>
  <p>
    195
  </p>
   - 
  <p>
    0
     
  </p>
</div>
`;
